tests/test_13_validate.py

```diff
@@ -146,3 +147,61 @@ def test_valid_address():
         assert valid_address("[2001:8003:5555:9999:555a:5555:c77:d5c5")
     with raises(NotValid):
         assert valid_address("[[2001:8003:5555:9999:555a:5555:c77:d5c5]")
+
+
+def test_valid_domain_name():
+    assert valid_domain_name("api.my-domain.com")
+    assert valid_domain_name("auth.admin.domain.com")
+    assert valid_domain_name("auth.domain.com")
+    assert valid_domain_name("auth.domain.com")
+    assert valid_domain_name("lk.domain.com:12")
+    assert valid_domain_name("lk.domain.com:12")
+    assert valid_domain_name("static.domain.xyz:12345")
```

Comment:
these are domains + ports; they are not just domains.

Reply:
yep it's dns name

Reply:
updated